### PR TITLE
43). Add topic files 135 to 138

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. It also fixes some article files.

Please merge #18439, #18541, #18580, #18589, #18673, #18700, #18711, #18781, and #18802 before merging this.

cc: @BillWagner @mairaw @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by mairaw** [comment]
Last set of merge conflicts to resolve for now @jameshkramer 😄

File: docs/standard/linq/use-annotations-transform-linq-xml-trees-xslt-style.md
Comment (mairaw) [inline docs/standard/linq/use-annotations-transform-linq-xml-trees-xslt-style.md:35]: I think this would be more clear

```suggestion
- One advantage to taking this approach is that, as you formulate queries, you're  always writing queries on the unmodified source tree. You don't need to worry about how modifications to the tree affect the queries that you're writing.
```

File: docs/standard/linq/namespaces-overview.md
Comment (mairaw) [inline docs/standard/linq/namespaces-overview.md:12]: I prefer the original here 
```suggestion
XML names are often a source of complexity in XML programming. An XML name consists of an XML namespace (also called an XML namespace URI) and a local name. An XML namespace is similar to a namespace in a .NET program. It enables you to uniquely qualify the names of elements and attributes to avoid name conflicts between various parts of an XML document. When you've declared an XML namespace, you can select a local name that only has to be unique within that namespace.
```

File: docs/standard/linq/serialize-datacontractserializer.md
Comment (mairaw) [inline docs/standard/linq/serialize-datacontractserializer.md:13]: I'd add some keywords to the introduction to help with SEO

```suggestion
This article shows an example that serializes and deserializes using <xref:System.Runtime.Serialization.DataContractSerializer> in C# and Visual Basic.
```

File: docs/standard/linq/serialize-xmlserializer.md
Comment (mairaw) [inline docs/standard/linq/serialize-xmlserializer.md:13]:
```suggestion
This article shows an example that serializes and deserializes using <xref:System.Xml.Serialization.XmlSerializer> in C# and Visual Basic.
```

File: docs/standard/linq/linq-xml-axes-overview.md
Comment (mairaw) [inline docs/standard/linq/linq-xml-axes-overview.md:9]: remove extra comma

```suggestion
After you've created an XML tree or loaded an XML document into an XML tree, you can query it to find elements and attributes and retrieve their values. You retrieve collections through the *axis methods*, also called *axes*. Some of the axes are methods in the <xref:System.Xml.Linq.XElement> and <xref:System.Xml.Linq.XDocument> classes that return <xref:System.Collections.Generic.IEnumerable%601> collections. Some of the axes are extension methods in the <xref:System.Xml.Linq.Extensions> class. The axes that are implemented as extension methods operate on collections and return collections.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good. A few suggestions...

**Review by mairaw — APPROVED** [review]
LGTM

**Comment by jameshkramer** [comment]
@mairaw @BillWagner @tfosmark @marteeleigh

Maira, thanks for getting these PRs merged. I'll have some changes to redirections this week. Then we test?

Jim

**Comment by mairaw** [comment]
Sounds good @jameshkramer. There are also some warnings showing up in #16969 after I merged the last PR, so you should take a look at that as well.

**Pull request merged**
